backend/sirius-components-emf/src/test/java/org/eclipse/sirius/components/emf/view/DynamicFormsTests.java

```diff
@@ -0,0 +1,113 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.emf.view;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.eclipse.emf.ecore.EClass;
+import org.eclipse.emf.ecore.EPackage;
+import org.eclipse.emf.ecore.EcoreFactory;
+import org.eclipse.emf.ecore.EcorePackage;
+import org.eclipse.sirius.components.core.api.IObjectService;
+import org.eclipse.sirius.components.emf.view.form.ViewFormDescriptionConverter;
+import org.eclipse.sirius.components.forms.AbstractWidget;
+import org.eclipse.sirius.components.forms.Form;
+import org.eclipse.sirius.components.forms.Page;
+import org.eclipse.sirius.components.forms.Textfield;
+import org.eclipse.sirius.components.forms.components.FormComponent;
+import org.eclipse.sirius.components.forms.components.FormComponentProps;
+import org.eclipse.sirius.components.forms.renderer.FormRenderer;
+import org.eclipse.sirius.components.representations.Element;
+import org.eclipse.sirius.components.representations.IRepresentationDescription;
+import org.eclipse.sirius.components.representations.VariableManager;
+import org.eclipse.sirius.components.view.FormDescription;
+import org.eclipse.sirius.components.view.TextfieldDescription;
+import org.eclipse.sirius.components.view.View;
+import org.eclipse.sirius.components.view.ViewFactory;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for dynamically defined forms.
+ *
+ * @author fbarbin
+ */
+public class DynamicFormsTests {
+
+    private static final String NAME_EXPRESSION = "aql:self.name"; //$NON-NLS-1$
```

Comment:
In unit test, you can use the `@SuppressWarnings("checkstyle:MultipleStringLiterals")` and not create constants for things like that. It does not make tests easier to read.

Reply:
Fixed.